db/migrate/20150120095012_create_categories.rb

```diff
@@ -2,6 +2,7 @@ class CreateCategories < ActiveRecord::Migration
   def change
     create_table :categories do |t|
       t.string :name
+      t.string :link
```

Comment:
What information (link) would this field store? The original link from Pinterest?